pages/parent.vue

```diff
@@ -1,13 +1,110 @@
+    "「新型コロナウイルス感染症にかかる相談窓口について」": "See the website: about COVID-19 telephone advisory center (Bureau of Social Welfare and Public Health)",
+    "その他": "Other information",
+    "詳細は、各学校からのお知らせ等をご確認ください。": "Make sure to check for any updates from the schools"
+  },
+  "zh-cn": {
+    "臨時休校中の新型コロナウイルス感染症対応についてのお願い": "关于临时停课期间的新冠肺炎防治对应",
+    "感染予防・健康管理": "预防感染与健康管理指南",
+    "不特定多数の人の集まる場所等への外出を避け、基本的に自宅で過ごしてください。": "〇 尽量不要出没于人多的场所，尽可能待在家中。",
+    "手洗い、咳エチケット等により、感染予防に努めてください。": "〇 勤洗手，咳嗽时注意卫生，以控制与预防感染。",
+    "【参考】感染症予防のための正しい手洗い方法（動画）": "【参考】预防感染的正确洗手方法（视频）",
+    "規則正しい生活を心がけ、日常の健康管理に十分気を付けてください。": "〇 尽可能保证规律作息，并请注意日常的健康管理。",
+    "感染症を疑う場合の対応": "当怀疑孩子感染新冠肺炎时",
+    "各保健所にご相談ください": "〇 如果有以下症状时，请联络就近的保健所：\n出现感冒的症状；\n持续 4 天发烧37.5℃以上；\n强烈的倦怠感；\n呼吸困难。",
+    "「新型コロナウイルス感染症にかかる相談窓口について」": "〇 「关于新型冠状病毒感染咨询窗口」（东京都福祉保健局）",
+    "その他": "其它",
+    "詳細は、各学校からのお知らせ等をご確認ください。": "〇 详情请参见各个学校的通知。"
+  },
+  "zh-tw": {
+    "臨時休校中の新型コロナウイルス感染症対応についてのお願い": "關於臨時停課期間的新型冠狀病毒肺炎防治對應",
+    "感染予防・健康管理": "預防感染與健康管理指南",
+    "不特定多数の人の集まる場所等への外出を避け、基本的に自宅で過ごしてください。": "〇 盡量不要出沒於人多的場所，盡可能的待在家裡。",
+    "手洗い、咳エチケット等により、感染予防に努めてください。": "〇 勤洗手，咳嗽時注意衛生，以控制與預防感染。",
+    "【参考】感染症予防のための正しい手洗い方法（動画）": "【參考】預防感染的正確洗手方法（影片）",
+    "規則正しい生活を心がけ、日常の健康管理に十分気を付けてください。": "〇 盡可能維持規律作息，並請注意日常的健康管理。",
+    "感染症を疑う場合の対応": "當懷疑罹患新型冠狀病毒肺炎時",
+    "各保健所にご相談ください": "〇 有感冒的症狀，或者是攝氏 37.5度以上的發燒持續4天以上，有強烈倦怠感，呼吸困難時，請和各地保健所連絡。",
+    "「新型コロナウイルス感染症にかかる相談窓口について」": "〇 「關於新型冠狀病毒感染諮詢窗口」（東京都福祉保建局）",
+    "その他": "其它",
+    "詳細は、各学校からのお知らせ等をご確認ください。": "〇 詳情請參閱各個學校的通知。"
+  },
+  "ko": {
+    "臨時休校中の新型コロナウイルス感染症対応についてのお願い": "임시 휴학 중의 코로나 19 대처를 위한 부탁 말씀",
+    "感染予防・健康管理": "감염 예방, 건강 관리",
+    "不特定多数の人の集まる場所等への外出を避け、基本的に自宅で過ごしてください。": "〇 불특정 다수의 사람이 모이는 장소 등으로의 외출을 삼가하고, 기본적으로 집 안에서 지내십시오. ",
+    "手洗い、咳エチケット等により、感染予防に努めてください。": "〇 손 씻기, 기침 예절 등으로, 감염 예방에 힘 써 주십시오. ",
+    "【参考】感染症予防のための正しい手洗い方法（動画）": "[참고자료] 감염증 예방을 위한 바른 손 씻기 방법 (동영상)",
+    "規則正しい生活を心がけ、日常の健康管理に十分気を付けてください。": "〇 규칙적인 생활을 유지하며 일상의 건강 관리에 충분히 주의하십시오.",
+    "感染症を疑う場合の対応": "감염이 의심될 경우의 대처 방법",
+    "各保健所にご相談ください": "〇 감기 증상, 4일 이상 지속되는 37.5도 이상의 발열, 강한 피로감(권태감), 호흡 곤란이 있을 경우, 각 보건소에 문의하십시오.",
+    "「新型コロナウイルス感染症にかかる相談窓口について」": "〇 \"신종 코로나 바이러스 감염 관련 전화 상담에 대해\" (도쿄도 복지 보건국)",
+    "その他": "其它",
+    "詳細は、各学校からのお知らせ等をご確認ください。": "〇 자세한 내용은 각 학교의 통지를 확인하시기 바랍니다."
+  },
+  "pt-BR": {
+    "臨時休校中の新型コロナウイルス感染症対応についてのお願い": "",
+    "感染予防・健康管理": "",
+    "不特定多数の人の集まる場所等への外出を避け、基本的に自宅で過ごしてください。": "",
+    "手洗い、咳エチケット等により、感染予防に努めてください。": "",
+    "【参考】感染症予防のための正しい手洗い方法（動画）": "",
+    "規則正しい生活を心がけ、日常の健康管理に十分気を付けてください。": "",
+    "感染症を疑う場合の対応": "",
+    "各保健所にご相談ください": "",
+    "「新型コロナウイルス感染症にかかる相談窓口について」": "",
+    "その他": "",
+    "詳細は、各学校からのお知らせ等をご確認ください。": ""
+  },
+  "ja-basic": {
+    "臨時休校中の新型コロナウイルス感染症対応についてのお願い": "とつぜん がっこうがやすみになっているあいだの、コロナにかからないようにするための おねがい",
+    "感染予防・健康管理": "コロナにかからないように・げんきでいるためにどうすればよいか",
+    "不特定多数の人の集まる場所等への外出を避け、基本的に自宅で過ごしてください。": "〇 たくさんのひとがあるまるところへいくのはやめましょう。なるべくいえにいてください。",
```

Comment:
[03/08 20:31 に変更](https://docs.google.com/spreadsheets/d/1aksSfWIXE_L3rFYFjHz9xtVhh0n7ealMQ0eEz8GGPlw/edit?disco=AAAAJLWr-Fc)されたようです
```suggestion
    "不特定多数の人の集まる場所等への外出を避け、基本的に自宅で過ごしてください。": "〇 ひとがいっぱいあつまるところに　いかないでください。なるべくいえにいてください。",
```